dm/proto/dmmaster.proto

```diff
@@ -78,6 +78,7 @@ message UpdateWorkerRelayConfigRequest {
 message StartTaskRequest {
     string task = 1; // task's configuration, yaml format
     repeated string sources = 2; // mysql source need to do start task, empty for all matched workers in deployment
+    bool RemoveMeta = 3; // whether to remove meta data for this task or not
```

Comment:
tiny comment, how about renaming `RemoveMeta` to `removeMeta` as we did for other fields?

Reply:
Addressed